pre-docs/bolt_installing_modules.md

```diff
@@ -0,0 +1,26 @@
+
+# Installing modules with Bolt
+
+Bolt uses a Puppetfile to specify which modules to install. The Puppetfile
+should be in the Boltdir, which can either be a project-specific directory
+named `Boltdir` at the root of your project, or a global directory in
+`$HOME/.puppetlabs/bolt`.
+
+The Puppetfile contains a list of modules and versions to install. If the
+modules have dependencies on other modules, those dependencies will need to be
+listed in the Puppetfile as well.
+
+By default, modules will be downloaded from the Puppet Forge. Modules can be
```

Comment:
Are we intentionally avoiding linking to the full Puppetfile docs after these examples?